ydb/core/persqueue/pqtablet/common/constants.h

```diff
@@ -13,4 +14,6 @@ static const ui32 MAX_BLOB_PART_SIZE = 500_KB;
 // Total time till kafka supportive partition deletion = AppData.KafkaProxyConfig.TransactionTimeoutMs + KAFKA_TRANSACTION_DELETE_DELAY_MS
 static const ui32 KAFKA_TRANSACTION_DELETE_DELAY_MS = TDuration::Hours(1).MilliSeconds(); // 1 hour;
 
-}
\ No newline at end of file
+static constexpr char TMP_REQUEST_MARKER[] = "__TMP__REQUEST__MARKER__";
```

Comment:
```suggestion
сonstexpr char TMP_REQUEST_MARKER[] = "__TMP__REQUEST__MARKER__";
```
В заголовочных файлах обычно же лучше без statiс, чтобы была одна копия переменной на всю программу, а не одна на каждый TU.
Впрочем, тут скорее всего эффект очень мал или даже отсутствует, если оптимизатор избавится от ODR-used